docs/metrics/extending-the-sdk/README.md

```diff
@@ -7,14 +7,68 @@
 
 ## Exporter
 
-TBD
+OpenTelemetry .NET SDK has provided the following built-in metric exporters:
+
+* [Console](../../../src/OpenTelemetry.Exporter.Console/README.md)
+* [InMemory](../../../src/OpenTelemetry.Exporter.InMemory/README.md)
+* [Prometheus](../../../src/OpenTelemetry.Exporter.Prometheus/README.md)
+* [OpenTelemetryProtocol](../../../src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md)
+
+Custom exporters can be implemented to send telemetry data to places which are
+not covered by the built-in exporters:
+
+* Exporters should derive from `OpenTelemetry.BaseExporter<Metric>` (which
+  belongs to the [OpenTelemetry](../../../src/OpenTelemetry/README.md) package)
+  and implement the `Export` method.
+* Exporters can optionally implement the `OnShutdown` method.
+* Exporters should not throw exceptions from `Export`, and
+  `OnShutdown`.
+* Exporters are responsible for any retry logic needed by the scenario. The SDK
+  does not implement any retry logic.
+* Exporters should avoid generating telemetry and causing live-loop, this can be
+  done via `OpenTelemetry.SuppressInstrumentationScope`.
+* Exporters receives a batch of `Metric`, and each `Metric`
+  can contain many `MetricPoint`s.
```

Comment:
It might be helpful to clarify if this could be "zero to many" or "1 or multiple".